### PR TITLE
Change Part-3 Exercise output probability calculation from softmax to torch.exp

### PR DESCRIPTION
**Pull request by aksungurlu**
In _"Part 3 Training Neural Networks (Exercise)"_ final part, output probabilities are calculated using **softmax** whereas in solution **torch.exp** is used. Torch.exp is the correct function since LogSoftmax is intended to be used.

Also comment is _"# Output of the network are logits, need to take softmax for probabilities"_ in both exercise and solution. I've changed it by _"# Output of the network are log of probabilities, need to calculate exp for probabilities"_

### CODE REVIEW

**Comment by mcleonard** [comment]
Thanks for the PR. There were some merge issues, so I fixed it in this commit: ae5248a4ae735fdb22a89b668261c629e3140d01
